lib/kontena/templates/haproxy/tcp_proxies.text.erb

```diff
@@ -0,0 +1,16 @@
+<% services.each do |service| %>
+<% if service.upstreams? && service.external_port %>
```

Comment:
Remove the `service.upstreams?` conditional to make this pass #29 for TCP services as well